src/plugins/condition/ConditionManager.js

```diff
@@ -131,58 +100,48 @@ export default class ConditionManager extends EventEmitter {
                 summary: ''
             };
         }
-        let conditionDomainObjectKeyString = this.openmct.objects.makeKeyString(conditionObj.identifier);
-        let newDomainObject = this.instantiate(conditionObj, conditionDomainObjectKeyString);
 
-        return newDomainObject.useCapability('adapter');
+        return conditionObj;
     }
 
-    addCondition(isDefault, index) {
-        this.createAndSaveConditionDomainObject(!!isDefault, index);
+    addCondition() {
+        this.createAndSaveCondition();
     }
 
-    cloneCondition(conditionConfigurationId, index) {
-        this.openmct.objects.get(conditionConfigurationId).then((conditionConfiguration) => {
-            this.createAndSaveConditionDomainObject(false, index, conditionConfiguration);
-        });
+    cloneCondition(conditionConfiguration, index) {
```

Comment:
This isn't working in my build. Condition cloned has default properties including name, output, match, and telemetry selection.

Reply:
Should be fixed now.